src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -265,39 +256,26 @@ export default {
 		},
 	},
 
-	mounted() {
-		EventBus.$on('new-group-conversation-dialog', this.showModalForItem)
-	},
-
-	destroyed() {
-		EventBus.$off('new-group-conversation-dialog', this.showModalForItem)
-	},
-
 	methods: {
-		showModal() {
-			this.modal = true
-		},
-
 		setIsAvatarEdited(value) {
 			this.isAvatarEdited = value
 		},
 
 		showModalForItem(item) {
 			if (item) {
 				// Preload the conversation name from group selection
-				this.conversation.displayName = item.label
+				this.newConversation.displayName = item.label
```

Comment:
When you rebase the branch later, you might face the conflict there, bc this line is on master already